openpype/host/interfaces.py

```diff
@@ -220,7 +220,7 @@ def work_root(self, session):
         Default implementation keeps workdir untouched.
 
         Warnings:
-            We must handle this modification with more sofisticated way because
+            We must handle this modification with more sophisticated way because
             this can't be called out of DCC so opening of last workfile
             (calculated before DCC is launched) is complicated. Also breaking
```

Comment:
```suggestion
            We must handle this modification with more sophisticated way
            because this can't be called out of DCC so opening of last workfile
            (calculated before DCC is launched) is complicated. Also breaking
```